app/views/helpers/menu.php

```diff
@@ -645,10 +647,7 @@ public function displayMenu(
 
         <li>
         <?php
-        echo $this->Html->div('loader-small loader', '', array(
-            'id' => '_'.$sentenceId.'_in_process',
-            'style' => 'display:none',
-        ));
+        echo "<md-progress-circular  md-diameter='11px' style='display: none;' id='_" . $sentenceId ."_in_process'></md-progress-circular>";
```

Comment:
I am not sure what code you are talking about? Are you saying I should remove the 'loader-small' class from 'ul' loader?